modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/paths/CP2NavigationMenu.path

```diff
@@ -95,6 +95,56 @@
 	<td>//a[contains(@class, 'font-weight-bold')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>SEARCH_FILTER</td>
+	<td>//div[contains(@class,'bg-neutral-1')]//div[@class="input-group-item"]//input[@placeholder="Search"]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>SEARCH_RESULT</td>
+	<td>//p[contains(@class, 'text-paragraph-sm') and contains(.,'result')]</td>
+	<td></td>
```

Comment:
@styller1 the locator here finding these classes are too fragile since these classNames are to volatile. Would you please change these locators to things other than the class names?